include/maths/CTools.h

```diff
@@ -22,6 +22,7 @@
 #include <boost/static_assert.hpp>
 
 #include <array>
+#include <cinttypes>
```

Comment:
Maybe cstdint would be better - see https://stackoverflow.com/a/7597070/3790930

Reply:
See [5d99c3e](https://github.com/elastic/ml-cpp/pull/1127/commits/5d99c3eec3b8d38f4d6ec429346cfebecf62f416).